src/pages/GameRedux.tsx

```diff
@@ -52,6 +53,9 @@ const GameRedux = ({ viewerIndex = FIRST_PLAYER_ID }: GameReduxProps) => {
   const playerState = useAppSelector(selectPlayerState); // Updated to use focused selector
   const gameConfig = useAppSelector(selectGameConfig);
   const gameProgress = useAppSelector(selectGameProgress);
+  const showWhitewashAnimation = useAppSelector(
+    (state: RootState) => state.game.uiState?.showWhitewashAnimation ?? false
```

Comment:
Create a new selector in selector.ts file called `selectShowWhiteWashAnimation`. Move this logic there.